content/en/docs/how-tos/mirroring-to-quay.md

```diff
@@ -40,6 +40,7 @@ repositories already have mirroring pipelines configured; each directory
 [here](https://github.com/openshift/release/tree/master/core-services/image-mirroring) corresponds to a repository.
 These directories contain mapping files that define tags on images in the target repository. New images may be submitted
 to mirror to existing organizations, or new ones.  When naming your new image, please follow the [naming guidelines](https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift/GUIDELINES.md).
+For OpenShift images, also consider consulting [the OpenShift mirroring approvers](https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift/OWNERS), to review your name choices, before sinking significant time into work that depends on that naming (like creating Git repositories or writing multiple Go packages).
```

Comment:
```suggestion
For OpenShift images(whether they will be included in the payload or not), also consider consulting [the OpenShift mirroring approvers](https://github.com/openshift/release/blob/master/core-services/image-mirroring/openshift/OWNERS), to review your name choices, before sinking significant time into work that depends on that naming (like creating Git repositories, requesting image repositories from Software Production, or writing multiple Go packages).
```

Reply:
Picked up, along with a pivot from "For OpenShift images" to "For `quay.io/openshift/...` images", because it's otherwise not all that clear what "OpenShift images" could mean if it extends beyond OpenShift release payloads, with 50def67 -> d6e0eb6.